### PR TITLE
chore: wrap momento metrics json in `momento` object for cloudwatch metric filters

### PR DESCRIPTION
**Pull request by anitarua**
So that #970 can incorporate MomentoMetricsMiddleware into the metric filters (e.g. `{$.MomentoMetricsMiddleware.status >= 0}`) to avoid clashing with other metrics customers may stream to a log group.

### CODE REVIEW

File: packages/client-sdk-nodejs/src/config/middleware/impl/experimental-metrics-middleware.ts [hunk 1]
```diff
@@ -24,50 +24,52 @@ const FIELD_NAMES: Array<string> = [
...
-  requestSize: number;
-  /**
-   * The size of the response body in bytes
-   */
-  responseSize: number;
-  /**
-   * The ID of the specific connection that made the request
-   */
-  connectionID: string;
+  MomentoMetricsMiddleware: {
```
Comment (pratik151192) [inline packages/client-sdk-nodejs/src/config/middleware/impl/experimental-metrics-middleware.ts:27]: not a strong opinion but I think `momento. numActiveRequestsAtStart` or `momento.duration` will be a cleaner UX than `MomentoMetricsMiddleware.duration`

**Review by pratik151192 — DISMISSED** [review]

File: packages/client-sdk-nodejs/src/config/middleware/impl/experimental-metrics-middleware.ts (hunk 1 above)
Comment (anitarua) [reply packages/client-sdk-nodejs/src/config/middleware/impl/experimental-metrics-middleware.ts]: I like that better too, updated

**Review by pratik151192 — APPROVED** [review]

**Review by cprice404 — APPROVED** [review]
oh yeah

**Comment by anitarua** [comment]
good call, the .NET update seems to be simpler

**Pull request merged**